### PR TITLE
add pagination in front

### PR DESCRIPTION
**Pull request by AllakhverdievRustam**

### CODE REVIEW

File: frontend-react-app/src/components/AddReceptionComponent/AddBlock.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const AddBlock = ({ setReceptions, setLengthReceptionArr }) => {
     const user = JSON.parse(localStorage.getItem('user'));
     const { authorization } = user;
 
-    axios.post('http://localhost:8000/createNewReception',
+    axios.post(`http://localhost:8000/createNewReception?limit=${limit}&offset=${offset}`,
```
Comment (usatovaDasha) [inline frontend-react-app/src/components/AddReceptionComponent/AddBlock.js:16]: why you send data in params, if you have body?

File: frontend-react-app/src/components/PaginationComponent/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const changePage = (element) => {
+    setOffset(element);
+    setUseEffectDo(true);
+  }
+
+  return (
+    <nav>
+      <ul className="pagination">
+        {
+          arrNunPage.map((element, index) => (
```
Comment (usatovaDasha) [inline frontend-react-app/src/components/PaginationComponent/Pagination.js:26]: you view users button "last", "first", "next", "prev"?

Comment (AllakhverdievRustam) [reply frontend-react-app/src/components/PaginationComponent/Pagination.js:26]: no, only numbers pages

File: frontend-react-app/src/components/AddReceptionComponent/AddBlock.js [hunk 3]
```diff
@@ -18,7 +18,9 @@ const AddBlock = ({ setReceptions, setLengthReceptionArr, offset, limit, setCoun
         nameUser: name,
         nameDoctor: doctor,
         date: date,
-        complaint: complaint
+        complaint: complaint,
```
Comment (usatovaDasha) [inline frontend-react-app/src/components/AddReceptionComponent/AddBlock.js]: complaint,
        limit,
        offset

File: frontend-react-app/src/components/PaginationComponent/Pagination.js [hunk 4]
```diff
@@ -10,20 +10,38 @@ const Pagination = ({ setOffset, limit, countAllReception, setUseEffectDo }) =>
...
+      setOffset(arrNunPage[arrNunPage.length - 1]);
+    } else setOffset(element);
     setUseEffectDo(true);
   }
 
   return (
     <nav>
       <ul className="pagination">
+        <li class="page-item">
+          <a onClick={() => changePage(-1)} class="pag-block page-link" href="#" aria-label="Previous">
```
Comment (usatovaDasha) [inline frontend-react-app/src/components/PaginationComponent/Pagination.js]: onClick={() => setOffset(0)}

File: frontend-react-app/src/components/PaginationComponent/Pagination.js [hunk 5]
```diff
@@ -10,20 +10,38 @@ const Pagination = ({ setOffset, limit, countAllReception, setUseEffectDo }) =>
...
+              <li key={`key-${index}`} className="page-item active">
+                <a onClick={() => changePage(element)} className="pag-block page-link">{element + 1}</a>
+              </li> :
+              <li key={`key-${index}`} className="page-item">
+                <a onClick={() => changePage(element)} className="pag-block page-link">{element + 1}</a>
+              </li>
           ))
         }
+        <li class="page-item">
+          <a onClick={() => changePage(-2)} class="pag-block page-link" href="#" aria-label="Next">
```
Comment (usatovaDasha) [inline frontend-react-app/src/components/PaginationComponent/Pagination.js]: higher

File: frontend-react-app/src/components/PaginationComponent/Pagination.js [hunk 6]
```diff
@@ -10,20 +10,38 @@ const Pagination = ({ setOffset, limit, countAllReception, setUseEffectDo }) =>
...
+            <span aria-hidden="true">&laquo;</span>
+          </a>
+        </li>
         {
           arrNunPage.map((element, index) => (
-            <li key={`key-${index}`} className="page-item">
-              <a onClick={() => changePage(element)} className="pag-block page-link">{element + 1}</a>
-            </li>
+            element === offset ?
+              <li key={`key-${index}`} className="page-item active">
```
Comment (usatovaDasha) [inline frontend-react-app/src/components/PaginationComponent/Pagination.js]: className=`page-item ${element === offset ? 'active'} : ''`

**Pull request merged**